README.md

```diff
@@ -1,2 +1,70 @@
-# portfolioskeleton1
-Mobile Portfolio Skeleton task 1
+
+# Project Name: Portfolio Skeleton Project
+
+> header and and headline section 
+Created new branch according to GitHub flow rules
+Added a .gitignore file
+setup a linters for html and css and made sure that files not relevant are ignored by git.
+Added a descriptive README file 
+Added a header and headline section in the index.html file  of the project 
+Added css styling to the html sections 
+
+## Built With
+- HTML and CSS
+- Visual Studio Code -Electron framework
+- Technologies used
+
+## Getting Started
+
+**This project include linters for HTML,CSS, and Open Source Document
+.**
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Node package Manager(NPM)
+Follow set up instructions from Node Official Website https://nodejs.org/en/download/
+Text Editor: VS-Code or any other Code editor 
+
+### Setup
+Simple Setup  folder
+  portfolioskeleton1
+  package.json
+  index.html
+  styles.css
+
```

Comment:
- [ ] It is important to note that this section, it is where you talk about procedures to follow from cloning your repository to running it locally in the machine. Please add the steps to follow when someone who is a developer like you want to use your repository 😉